packages/core/src/domain/document/upload.ts

```diff
@@ -18,3 +19,16 @@ export function createUploadMetadataFilePath(
   const uploadFilePath = createUploadFilePath(cxId, patientId, docName);
   return `${uploadFilePath}_metadata.xml`;
 }
+
+export function createAttachmentUploadPath({
+  fileName,
+  ownId,
```

Comment:
bit a strange name for a param -- `id` or `attachmentId` might make more sense...

Reply:
fixed